taranis/__main__.py

```diff
@@ -184,29 +190,41 @@ def taranis_cli(verbose, log_file):
     default="Genus",
     help="Use genus-specific BLAST databases for Prokka schema genes annotation (needs --genus). Default is False.",
 )
+@click.option(
+    "--cpus",
+    required=False,
+    multiple=False,
+    type=int,
+    default=1,
+    help="Number of cpus used for execution",
```

Comment:
cpus or threads finally?

Reply:
I pretty sure handle by CPU would be faster than using threads, but  I want to confirm by checking in HPC

Reply:
I test using thread on my laptop and execution time is the same for 13 locus and 5 cpus in parameter spent 2.21 minutes.
However in cluster could have different results.
The image show the execution using threads and second cpus
![image](https://github.com/BU-ISCIII/taranis/assets/24755054/6ca69866-099e-46d6-bef7-09bfbc09c252)

Reply:
As we already discussed I think its more conceptual than anything...but you can leave it that way and we can revisit this later once we've made all the tests in the HPC. 
I would go for threads just for make it easier for the user, as most of us now how many threads do we have, without knowing if they are physical cpus or threads